### DISCUSSION TITLE
TypeScript course

### DISCUSSION THREAD

**Discussion by nik-rev** [TheOdinProject/curriculum#27346] [Ideas]
Hey, this is just a general idea I'm throwing out there, but I see that many job postings require TypeScript knowledge. So what do we think if there was a dedicated TS course after the NodeJS path? 

This could take a very long time to get the course out, maybe years. But I think that in the future a TS course from TOP would be valuable. And it would be beneficial to get started working on it early. 

I don't know anything about typescript, I just know that the job market favors it. If there was a TS course it would help people become more employable in the future.

**Comment by KevinMulhern** [comment]
Thanks for the suggestion @nikitarevenco, we have an existing discussion about this. Would you mind posting your rationale on that thread please? https://github.com/TheOdinProject/curriculum/discussions/27208
